[Brats17_TCIA_640_1_t2_unbiased.nii.gz, Brats17_TCIA_608_1_t2_unbiased.nii.gz, Brats17_TCIA_620_1_t2_unbiased.nii.gz, Brats17_TCIA_149_1_t2_unbiased.nii.gz, Brats17_TCIA_606_1_t2_unbiased.nii.gz, Brats17_CBICA_AAG_1_t2_unbiased.nii.gz, Brats17_CBICA_AQO_1_t2_unbiased.nii.gz, Brats17_CBICA_AQR_1_t2_unbiased.nii.gz, Brats17_TCIA_351_1_t2_unbiased.nii.gz, Brats17_CBICA_AVV_1_t2_unbiased.nii.gz]
 Vae model:  camcan_400_Aug_6_100  NN model:  10subj_3lr_10steps_100K_2fch_BCE_1_100
Using device: cuda:0
/scratch_net/biwidl214/jonatank/anaconda3/envs/JKMT/lib/python3.7/site-packages/torch/serialization.py:593: SourceChangeWarning: source code of class 'networks.vae_bilinear_conv.ResBlock_Down' has changed. you can retrieve the original source code by accessing the object's source attribute or set `torch.nn.Module.dump_patches = True` and use the patch tool to revert the changes.
  warnings.warn(msg, SourceChangeWarning)
/scratch_net/biwidl214/jonatank/anaconda3/envs/JKMT/lib/python3.7/site-packages/torch/serialization.py:593: SourceChangeWarning: source code of class 'networks.vae_bilinear_conv.ResBlock_Up' has changed. you can retrieve the original source code by accessing the object's source attribute or set `torch.nn.Module.dump_patches = True` and use the patch tool to revert the changes.
  warnings.warn(msg, SourceChangeWarning)
Traceback (most recent call last):
  File "restore_MAP_NN.py", line 67, in <module>
    net = torch.load(path, map_location=torch.device(device))
  File "/scratch_net/biwidl214/jonatank/anaconda3/envs/JKMT/lib/python3.7/site-packages/torch/serialization.py", line 525, in load
    with _open_file_like(f, 'rb') as opened_file:
  File "/scratch_net/biwidl214/jonatank/anaconda3/envs/JKMT/lib/python3.7/site-packages/torch/serialization.py", line 212, in _open_file_like
    return _open_file(name_or_buffer, mode)
  File "/scratch_net/biwidl214/jonatank/anaconda3/envs/JKMT/lib/python3.7/site-packages/torch/serialization.py", line 193, in __init__
    super(_open_file, self).__init__(open(name, mode))
FileNotFoundError: [Errno 2] No such file or directory: '/scratch_net/biwidl214/jonatank/logs/restore/10subj_3lr_10steps_100K_2fch_BCE_1_100.pth'
[Brats17_TCIA_208_1_t2_unbiased.nii.gz]
 Vae model:  camcan_400_Aug_6_100  NN model:  10subj_3lr_10steps_100K_2fch_BCE_1_100
Using device: cuda:0
/scratch_net/biwidl214/jonatank/anaconda3/envs/JKMT/lib/python3.7/site-packages/torch/serialization.py:593: SourceChangeWarning: source code of class 'networks.vae_bilinear_conv.ResBlock_Down' has changed. you can retrieve the original source code by accessing the object's source attribute or set `torch.nn.Module.dump_patches = True` and use the patch tool to revert the changes.
  warnings.warn(msg, SourceChangeWarning)
/scratch_net/biwidl214/jonatank/anaconda3/envs/JKMT/lib/python3.7/site-packages/torch/serialization.py:593: SourceChangeWarning: source code of class 'networks.vae_bilinear_conv.ResBlock_Up' has changed. you can retrieve the original source code by accessing the object's source attribute or set `torch.nn.Module.dump_patches = True` and use the patch tool to revert the changes.
  warnings.warn(msg, SourceChangeWarning)
Traceback (most recent call last):
  File "restore_MAP_NN.py", line 67, in <module>
    net = torch.load(path, map_location=torch.device(device))
  File "/scratch_net/biwidl214/jonatank/anaconda3/envs/JKMT/lib/python3.7/site-packages/torch/serialization.py", line 525, in load
    with _open_file_like(f, 'rb') as opened_file:
  File "/scratch_net/biwidl214/jonatank/anaconda3/envs/JKMT/lib/python3.7/site-packages/torch/serialization.py", line 212, in _open_file_like
    return _open_file(name_or_buffer, mode)
  File "/scratch_net/biwidl214/jonatank/anaconda3/envs/JKMT/lib/python3.7/site-packages/torch/serialization.py", line 193, in __init__
    super(_open_file, self).__init__(open(name, mode))
FileNotFoundError: [Errno 2] No such file or directory: '/scratch_net/biwidl214/jonatank/logs/restore/10subj_3lr_10steps_100K_2fch_BCE_1_100.pth'
[Brats17_TCIA_451_1_t2_unbiased.nii.gz, Brats17_TCIA_328_1_t2_unbiased.nii.gz, Brats17_TCIA_150_1_t2_unbiased.nii.gz, Brats17_2013_11_1_t2_unbiased.nii.gz, Brats17_TCIA_319_1_t2_unbiased.nii.gz, Brats17_TCIA_111_1_t2_unbiased.nii.gz, Brats17_CBICA_AVG_1_t2_unbiased.nii.gz, Brats17_TCIA_254_1_t2_unbiased.nii.gz, Brats17_TCIA_274_1_t2_unbiased.nii.gz, Brats17_CBICA_AYA_1_t2_unbiased.nii.gz]
 Vae model:  camcan_400_Aug_6_100  NN model:  10subj_3lr_10steps_100K_2fch_BCE_2_100
Using device: cuda:0
/scratch_net/biwidl214/jonatank/anaconda3/envs/JKMT/lib/python3.7/site-packages/torch/serialization.py:593: SourceChangeWarning: source code of class 'networks.vae_bilinear_conv.ResBlock_Down' has changed. you can retrieve the original source code by accessing the object's source attribute or set `torch.nn.Module.dump_patches = True` and use the patch tool to revert the changes.
  warnings.warn(msg, SourceChangeWarning)
/scratch_net/biwidl214/jonatank/anaconda3/envs/JKMT/lib/python3.7/site-packages/torch/serialization.py:593: SourceChangeWarning: source code of class 'networks.vae_bilinear_conv.ResBlock_Up' has changed. you can retrieve the original source code by accessing the object's source attribute or set `torch.nn.Module.dump_patches = True` and use the patch tool to revert the changes.
  warnings.warn(msg, SourceChangeWarning)
Traceback (most recent call last):
  File "restore_MAP_NN.py", line 67, in <module>
    net = torch.load(path, map_location=torch.device(device))
  File "/scratch_net/biwidl214/jonatank/anaconda3/envs/JKMT/lib/python3.7/site-packages/torch/serialization.py", line 525, in load
    with _open_file_like(f, 'rb') as opened_file:
  File "/scratch_net/biwidl214/jonatank/anaconda3/envs/JKMT/lib/python3.7/site-packages/torch/serialization.py", line 212, in _open_file_like
    return _open_file(name_or_buffer, mode)
  File "/scratch_net/biwidl214/jonatank/anaconda3/envs/JKMT/lib/python3.7/site-packages/torch/serialization.py", line 193, in __init__
    super(_open_file, self).__init__(open(name, mode))
FileNotFoundError: [Errno 2] No such file or directory: '/scratch_net/biwidl214/jonatank/logs/restore/10subj_3lr_10steps_100K_2fch_BCE_2_100.pth'
[Brats17_TCIA_192_1_t2_unbiased.nii.gz]
 Vae model:  camcan_400_Aug_6_100  NN model:  10subj_3lr_10steps_100K_2fch_BCE_2_100
Using device: cuda:0
/scratch_net/biwidl214/jonatank/anaconda3/envs/JKMT/lib/python3.7/site-packages/torch/serialization.py:593: SourceChangeWarning: source code of class 'networks.vae_bilinear_conv.ResBlock_Down' has changed. you can retrieve the original source code by accessing the object's source attribute or set `torch.nn.Module.dump_patches = True` and use the patch tool to revert the changes.
  warnings.warn(msg, SourceChangeWarning)
/scratch_net/biwidl214/jonatank/anaconda3/envs/JKMT/lib/python3.7/site-packages/torch/serialization.py:593: SourceChangeWarning: source code of class 'networks.vae_bilinear_conv.ResBlock_Up' has changed. you can retrieve the original source code by accessing the object's source attribute or set `torch.nn.Module.dump_patches = True` and use the patch tool to revert the changes.
  warnings.warn(msg, SourceChangeWarning)
Traceback (most recent call last):
  File "restore_MAP_NN.py", line 67, in <module>
    net = torch.load(path, map_location=torch.device(device))
  File "/scratch_net/biwidl214/jonatank/anaconda3/envs/JKMT/lib/python3.7/site-packages/torch/serialization.py", line 525, in load
    with _open_file_like(f, 'rb') as opened_file:
  File "/scratch_net/biwidl214/jonatank/anaconda3/envs/JKMT/lib/python3.7/site-packages/torch/serialization.py", line 212, in _open_file_like
    return _open_file(name_or_buffer, mode)
  File "/scratch_net/biwidl214/jonatank/anaconda3/envs/JKMT/lib/python3.7/site-packages/torch/serialization.py", line 193, in __init__
    super(_open_file, self).__init__(open(name, mode))
FileNotFoundError: [Errno 2] No such file or directory: '/scratch_net/biwidl214/jonatank/logs/restore/10subj_3lr_10steps_100K_2fch_BCE_2_100.pth'
